recipe-client-addon/content/about-studies/shield-studies.js

```diff
@@ -0,0 +1,151 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+"use strict";
+
+window.ShieldStudies = class ShieldStudies extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      learnMoreHref: null,
+    };
+  }
+
+  componentDidMount() {
+    remoteValues.ShieldLearnMoreHref.subscribe(this);
+  }
+
+  componentWillUnmount() {
+    remoteValues.ShieldLearnMoreHref.unsubscribe(this);
+  }
+
+  receiveRemoteValue(name, value) {
+    this.setState({
+      learnMoreHref: value,
+    });
+  }
+
+  render() {
+    return (
+      r("div", {},
+        r(InfoBox, {},
+          r("span", {}, "What's this? Firefox may install and run studies from time to time."),
```

Comment:
I checked with flod, and because users only get to this page by typing it into the address bar, and because we won't be running studies outside of English locales for a while anyway, he's okay shipping this without anything hiding the page for non-en-* locales. I didn't get an "official" signoff though.

The plan is to ship l10n support ASAP after we ship the initial version.